modprobe.conf/modprobe.conf.fs-blacklist

```diff
@@ -0,0 +1,32 @@
+# These file systems are not used on most systems but the kernel
+# will still happily load the modules for them when presented with
+# a potentially malicious device like a USB stick.  Many of these
+# have not been maintained except for in-kernel API compatibility
+# in many years, have not been audited, and may have security vulnerabilities.
+#
+# The following list only specifies local file systems since those are the
+# only ones that will be detected automatically by mount(8).
+#
+# Enable at your own risk.
+#
+blacklist adfs
+blacklist affs
+blacklist bfs
+blacklist befs
+blacklist cramfs
+blacklist efs
+blacklist erofs
+blacklist exofs
+blacklist freevxfs
+blacklist f2fs
+blacklist hfs
+blacklist hpfs
+blacklist jffs2
+blacklist jfs
+blacklist minix
+blacklist nilfs2
+blacklist qnx4
+blacklist qnx6
+blacklist sysv
+blacklist ubifs
```

Comment:
Same as for jffs2. But UBIFS is actively developed and maintained.
I can understand that OpenSUSE blacklists flash filesystems because you need them only on embedded
devices with raw flash. But please don't claim that they are unmaintained/vulnerable.

Reply:
The same applies to ubifs.  But your more general point is well taken. You're right. Martin Wilck has a newer version of this that creates one file per blacklist entry and has a comment at the top of each.  Currently it uses the phrasing " because it isn't actively supported by SUSE, not well maintained, and may have security vulnerabilites."

It would make more sense to change the phrasing to "or" instead of "and."